scripts/generate_syncset.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/python
+import os
+import sys
+import yaml 
+import argparse
+
+def get_yaml_all(filename):
+    with open(filename,'r') as input_file:
+        return list(yaml.load_all(input_file))
+
+def get_yaml(filename):
+    with open(filename,'r') as input_file:
+        return yaml.safe_load(input_file)
+
+def get_all_yaml_files(path):
+    file_paths = []
+    for r,_,f in os.walk(path):
+        for file in f:
+            if '.yml' in file:
+                file_paths.append(os.path.join(r,file))
+        for file in f: 
+            if '.yaml' in file:
+                file_paths.append(os.path.join(r,file))
+    return file_paths
+
+def get_all_yaml_obj(file_paths):
+    yaml_objs = []
+    for file in file_paths:
+        objects = get_yaml_all(file)
+        for obj in objects:
+            yaml_objs.append(obj)
+    return yaml_objs
+
+
+if __name__ == '__main__':
+    #Argument parser
+    parser = argparse.ArgumentParser(description="selectorsyncset generation tool")
```

Comment:
Adding `, usage='%(prog)s [options]'` to `argparse.ArgumentParser()` would print the name of the file in the help usage like `usage: generate_syncset.py [options]`